data/modules/Debug/DebugCommodityPrices.lua

```diff
@@ -175,24 +160,27 @@ local scan_systems = function(dist)
 			-- table.sort(comm, function (a,b) return a:GetName() > b:GetName() end)
 
 			-- include goods if legal, or if we've chosen to include it
-			local include = include_illegal or sys:IsCommodityLegal(comm.name)
+			local include = sys:IsCommodityLegal(comm.name)
+			if include_illegal then
+				include = true
+			end
```

Comment:
Note: this does the same thing in more lines.

Reply:
Yes, I re-wrote it like that, (temporarily, I thought) when debugging. I'll revert it.